web/libs/editor/src/components/SidePanels/DetailsPanel/RegionDetails.tsx

```diff
@@ -67,6 +67,16 @@ const ResultItem: FC<{ result: any }> = observer(({ result }) => {
         </Elem>
       );
     }
+    if (type === "taxonomy") {
+      return (
+        <Elem name="result">
+          <Text>Taxonomy: </Text>
+          <Elem name="value">
+            <ChoicesResult mainValue={mainValue.map((v: string[]) => v.join("/"))} />
```

Comment:
if we're mapping this array - it might be good to `useMemo` to memoize the value so we dont have to constantly redo this on render

Reply:
This component is not going to be rerendered often, so no need in `useMemo()`